vlsi/sim.mk

```diff
@@ -11,6 +11,8 @@ redo-sim-syn: override HAMMER_EXTRA_ARGS += -p $(SIM_CONF)
 redo-sim-syn: override HAMMER_SIM_RUN_DIR = sim-syn-rundir
 redo-sim-syn-debug: $(SIM_DEBUG_CONF) redo-sim-syn
 redo-sim-syn-debug: override HAMMER_EXTRA_ARGS += -p $(SIM_DEBUG_CONF)
+redo-sim-syn-timing-debug: $(SIM_TIMING_CONF) redo-sim-par-debug
```

Comment:
The token on this line should be `redo-sim-syn-debug`

Reply:
Oops!